src/plugins/tables/tables.js

```diff
@@ -84,7 +84,8 @@ var componentName = "wb-tables",
 				},
 				complete: function() {
 					var $elm = $( "#" + elmId ),
-						dataTableExt = $.fn.dataTableExt;
+						dataTableExt = $.fn.dataTableExt,
+						userDefaults = wb.getData( $elm, componentName );
```

Comment:
rename "userDefaults" for "settings". That is for consistency naming between plugin. For reference check the step 3 in the [hello world example](https://wet-boew.github.io/wet-boew/demos/helloworld/helloworld-en.html)